lib/internal/modules/esm/hooks.js

```diff
@@ -87,45 +86,40 @@ let importMetaInitializer;
 // [2] `validate...()`s throw the wrong error
 
 class Hooks {
-  #chains = {
```

Comment:
Why remove these? If you need them to start out empty, you can still leave the fields (`resolve` etc) and then also the code docs

Reply:
Ah, yes. I figured that :) I think you can still leave everything else here and just initialise the lists as empty 🙂

Reply:
Ah yes true 😄 I can do that. Part of me considered that hard-coding the hook names in a bunch of places might be less clean… but it's more clear and that works for me 🎉